README.md

```diff
@@ -3843,6 +3843,7 @@ also known as
 cat .git/config
 is a useful command by the way
 
+<<<<<<< HEAD
```

Comment:
hmmm, this seems like the peace form a conflict resolving, should be deleted pls

Reply:
Fixed